showcase-app/test-cases.md

```diff
@@ -0,0 +1,91 @@
+# End-to-End Test Documentation
+
+This document provides an overview of the end-to-end tests for the plugin.
+
+## Test Cases
+
+### 1. Page Has Title And Headlines
+
+- **Description**: Tests if the main page has the correct title and header.
+- **Test Steps**:
+  - Open the main page.
+  - Check if the page title matches "Solid Auth Showcase".
+  - Check if the page main headline matches "COOL SOLID APP".
+  - Check if the page sub headline matches "Log in with your Solid Identity".
+
+### 2. Page Shows Error Message When Extension Is Not Available
+
+- **Description**: Tests if an error message is displayed when no connection can be made to the Solid Identity Extension.
+- **Test Steps**:
+  - Assert the extension is not available. 
+  - Open the main page.
+  - Verify that a message indicates that no connection can be made to the extension.
+
+### 3. Default State With No Active Profile
+
+- **Description**: Tests if the default state (no active profile) of the page is correct.
+- **Test Steps**:
+  - Open the main page.
+  - Assert the extension doesn't return an active profile.
+  - Verify that the page does not show an active profile.
+
+### 4. Switching Profiles Activates the Correct Profile
+
+- **Description**: Tests if switching between profiles correctly activates the correct profile in the app.
+- **Test Steps**:
+  - Setup 3 profiles in the extension.
+  - Switch between the profiles and verify that the correct profile is always activated on the main page.
+
+### 5. Removing Profile Inside the Extension Deactivates the Profile
+
+- **Description**: Tests if logging out inside the extension deactivates the profile.
```

Comment:
The user can't log out of a profile considering that never log in with their WebID/IDP.

Reply:
I've reworded it to "remove profile" instead of logging out